### ISSUE TITLE
各エポックでデータをシャッフルすべきか？

### ISSUE THREAD

**Issue by RioRioWeb** [RioRioWeb/CLIP-Text-Decoder#12]
現状
`train_dataloader = DataLoader(dataset, batch_size=batch_size, drop_last=True)`

シャッフルするなら、
shuffle=Trueを引数に追加する。
